integration/sandbox_run_rollback_test.go

```diff
@@ -36,9 +37,11 @@ import (
 	"github.com/stretchr/testify/require"
 	criapiv1 "k8s.io/cri-api/pkg/apis/runtime/v1"
 
+	"github.com/containerd/typeurl"
```

Comment:
looks like this one accidentally moved 😁

Reply:
Fixed